resolver/resolver.cc

```diff
@@ -2485,11 +2485,14 @@ class ResolveSignaturesWalk {
             int j = 0;
             for (auto spec : sigParams) {
                 auto param = defParams[j];
-                auto sname = spec.name.show(ctx);
-                auto dname = param->localVariable._name.show(ctx);
-                if (sname != dname) {
+                auto sname = spec.name;
+                auto dname = param->localVariable._name;
+                // Common case: NameRefs match. Uncommon case: NameRefs don't match but their show strings are
+                // identical.
```

Comment:
I actually don't even know if namerefs can be != and show be ==, but I don't want to break anything.

Reply:
How hard would it be to generate a coverage report? It would be interesting to see if any of our test suite triggers this case.

Reply:
I can just insert an ENFORCE and see what happens.

Reply:
it, uh, it never triggers that case.

Reply:
I'll insert a comment about it.

Reply:
Nice!